apps/judicial-system/web/src/routes/Court/InvestigationRequest/Ruling/RulingForm.tsx

```diff
@@ -80,6 +81,47 @@ const RulingForm: React.FC<Props> = (props) => {
             </AccordionItem>
           </Accordion>
         </Box>
+        <Box component="section" marginBottom={5}>
+          <Box marginBottom={3}>
+            <Text as="h3" variant="h3">
+              {formatMessage(m.sections.introduction.title)}
+            </Text>
+          </Box>
+          <Input
+            data-testid="introduction"
+            name="introduction"
+            label={formatMessage(m.sections.introduction.label)}
+            value={workingCase.introduction || ''}
+            placeholder={formatMessage(m.sections.introduction.placeholder)}
+            onChange={(event) =>
+              removeTabsValidateAndSet(
+                'introduction',
+                event.target.value,
+                ['empty'],
+                workingCase,
+                setWorkingCase,
+                introductionEM,
+                setIntroductionEM,
+              )
+            }
+            onBlur={(event) =>
+              validateAndSendToServer(
+                'introduction',
+                event.target.value,
+                ['empty'],
+                workingCase,
+                updateCase,
+                setIntroductionEM,
+              )
+            }
+            errorMessage={introductionEM}
+            hasError={introductionEM !== ''}
+            textarea
+            rows={7}
+            autoExpand={{ on: true, maxHeight: 300 }}
+            required
```

Comment:
Var ekki talað um að þetta myndi vera optional svæði? er ég að fara með rangð mál?

Reply:
Good question, but I don't think so. Until now, we have added a fixed text to the PDF. This will replace it, but there should always be some intro to the ruling.

Reply:
It's marked as required in the design 🤷